corehq/form_processor/models.py

```diff
@@ -722,6 +722,23 @@ def get_obj_by_id(cls, case_id):
         from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
         return CaseAccessorSQL.get_case(case_id)
 
+    @property
+    @memoized
+    def parent(self):
+        """
+        Returns the parent case if one exists, else None.
+        NOTE: This property should only return the first parent in the list
+        of indices. If for some reason your use case creates more than one,
+        please write/use a different property.
```

Comment:
I think it's save to assume the identifier will be parent. There are cases
where that isn't true but they are usually custom.